Dockerfile

```diff
@@ -1,5 +1,5 @@
 # ---- Base Image ----
-ARG base=hmctspublic.azurecr.io/base/node:12-stretch-slim
+ARG base=hmctspublic.azurecr.io/base/node/alpine-lts-10:10-alpine
```

Comment:
```suggestion
ARG base=hmctspublic.azurecr.io/base/node:12-alpine
```

Reply:
Had realised this and updated to point to the correct image before I saw your comment. The below is the correct one, right?